src/main/java/me/qoomon/gradle/gitversioning/GitVersioningPluginExtension.java

```diff
@@ -567,6 +567,8 @@ private Map<String, Supplier<String>> generateGlobalFormatPlaceholderMap(GitSitu
             return matcher;
         });
 
+        final Lazy<String> labelAsInteger = Lazy.by(() -> extractLabelNumber(descriptionTagVersionComponents));
```

Comment:
I don't think we need this at all as well as we don't need `extractLabelNumber`. We could just use `placeholderMap.get("describe.tag.version.label").get()` and refactor `increaseStringNumberBy` to treat empty string as `0`

Reply:
Done - see latest commit. Nice spot!